tools/lint.mjs

```diff
@@ -341,14 +341,17 @@ for (const match of source.matchAll(/\|(\w+)\|\.{{(\w+)\/.*?}}/g)) {
   });
 }
 
+// TODO: Generate this from the IDL itself.
+const dictionaries = ['MLOperandDescriptor', 'MLContextLostInfo'];
+
 // Ensure JS objects are created with explicit realm
 for (const match of text.matchAll(/ a new promise\b(?! in realm)/g)) {
   error(`Promise creation must specify realm: ${format(match)}`);
 }
 for (const match of text.matchAll(/ be a new ([A-Z]\w+)\b(?! in realm)/g)) {
   const type = match[1];
   // Dictionaries are just maps, so they don't need a realm.
-  if (type === 'MLOperandDescriptor')
+  if (dictionaries.includes(type))
```

Comment:
```suggestion
  if (dictionaryTypes.includes(type))
```
(minor naming) Makes it slightly clearer that it it's a dictionary of types that are dictionaries :b (otherwise `dictionaries` could be anything).

Reply:
Done in 5fe8c9e